src/workloads/networking/TransportLayerConnectTiming.yml

```diff
@@ -0,0 +1,34 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/security"
+Description: >-
+    Invoke replSetTestEgress command on replica set members.
+    Command synchronously establishes temporary connections between cluster nodes.
+
+Clients:
+  Default:
+    QueryOptions:
+      maxPoolSize: 10
+      socketTimeoutMS: 5000
+      connectTimeoutMS: 5000
+Actors:
+- Name: TestEgress
+  Type: RunCommand
+  Threads: 1
+  Phases:
+  - Phase: 0
+    Duration: 5 minutes
+    GlobalRate: 1 per 1 microsecond
```

Comment:
How long does a connection take to create? More specifically is it greater than or less than a microsecond. If it's less we don't need to rate limit it at all and could let the thread run as fast as it can.

Reply:
At minimum 200ms using the configuration we have (with induced latency), but even without that extra latency I'd be amazed if these connection establishments took less than 1 MILLIsecond, let alone a microsecond.  May I assume your use of "If it's less" was a typo, and you're saying that in this case I can remove the line completely and just let 'er ride?

Reply:
> At minimum 200ms using the configuration we have (with induced latency), but even without that extra latency I'd be amazed if these connection establishments took less than 1 MILLIsecond, let alone a microsecond. May I assume your use of "If it's less" was a typo, and you're saying that in this case I can remove the line completely and just let 'er ride?

Yeah -- just delete line 20 and let her ride. It shouldn't be doing anything with those times. If your intent was to put delay between connection creation, that would be a different thing. 

@rtimmons can you just confirm I haven't said something crazy? I haven't touched Genny in a bit.